wordpress-mobile/gutenberg-mobile
# Add missing heading options

Fixes #1261 

As discussed with @iamthomasbishop on the ticket above we decided to implement the missing heading options by just adding them to the format toolbar.

Related GB PR: https://github.com/WordPress/gutenberg/pull/17533

![Simulator Screen Shot - iPhone 11 - 2019-09-23 at 22 08 31](https://user-images.githubusercontent.com/651601/65462939-b7901800-de4e-11e9-9be3-d2a40670b4e8.png)


To test:
 - Start the demo app
 - Check that the heading block has the H4, H5, H6 buttons when selected
 - Use the option and check that the HTML is correctly generated.

Update release notes:
